protocol/messenger.go

```diff
@@ -4202,6 +4209,7 @@ func (m *Messenger) markMessagesSeenImpl(chatID string, ids []string) (uint64, u
 	return count, countWithMentions, nil
 }
 
+// Deprecated: Use MarkMessagesRead instead
```

Comment:
I believe you should use the deprecated pragma instead, because a comment is not enough to help future developers

Example :

```
func (m *Messenger) MarkMessagesSeen(chatID string, ids []string) (uint64, uint64, []*ActivityCenterNotification, error) {.deprecated: "use MarkMessagesRead instead".} 
```